kubernetes/restic/maintenance-wrapper.sh

```
@@ -0,0 +1,67 @@
+fi
+
+uuid=$(cat /tmp/uuid)
+hc_url="https://hc-ping.com/${HEALTHCHECK_PING_KEY}/kube-restic-${profile}-maintenance"
+
+# Track state for cleanup
+rc=1
+failed_command=""
+log_file="/tmp/maintenance-$$.log"
+
+# Tee all output to log file for error capture
+exec > >(tee -a "$log_file") 2>&1
```

Shellcheck (0.11.0)

[warning] In POSIX sh, process substitution is undefined.

(SC3001)